index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE HTML>
+<html>
+    <head>
+        <meta charset="UTF-8">
```

Comment:
Please make sure to include a viewport meta tag in the head, so that your site will be mobile responsive:

`<meta name="viewport" content="device-width, initial-scale=1">`